frontend/src/components/Forms/Patient/PatientAddProfileForm.tsx

```diff
@@ -202,23 +198,23 @@ const PatientBasicDetails = () => {
 								label="Email address"
 								placeholder="Email address"
 							/>
-							<InputField
-								name="ptMobilePhone"
-								type="text"
-								label="Mobile phone number"
-								placeholder="Mobile phone number"
-							/>
 						</SimpleGrid>
 						<SimpleGrid
 							columns={[1, 1, 1, 1, 1, 2]}
 							spacing={[1, 1, 1, 1, 1, 4]}
 						>
 							<InputField
+								name="ptMobilePhone"
+								type="text"
+								label="Mobile phone number"
+								placeholder="Mobile phone number"
+							/>
+							{/* <InputField
 								name="ptHomePhone"
 								type="text"
 								label="Home phone number"
 								placeholder="Home phone number"
-							/>
+							/> */}
```

Comment:
@ThanhNguyenn

This completely removes the home phone input field for the patient's add profile details form. What was the rationale behind this change?

Reply:
Yes Glaiza as I was trying to adding some testing field to that form to ensure the logic flow of the application. I will put this back in the next PR.

Reply:
I see, thanks for clarifying. I'll resolve this one as well for now so you can merge your PR.